doc/modules/linear_model.rst

```diff
@@ -266,6 +266,11 @@ They also tend to break when the problem is badly conditioned
 
   * :ref:`sphx_glr_auto_examples_linear_model_plot_lasso_model_selection.py`
 
+.. note:: **Comparison with the regularization parameter of SVM**
```

Comment:
I think @amueller is saying that the current description here is not specific enough, or not very helpful to the reader.